Dynatrace/dynatrace-operator
# Display warning if ONEAGENT_INSTALLER_SCRIPT_URL or ONEAGENT_INSTALLER_TOKEN env vars are used

## Description
Using ONEAGENT_INSTALLER_SCRIPT_URL and ONEAGENT_INSTALLER_TOKEN  is an indication that an unsupported OneAgent image type is used. In this case a warning shall be displayed.

## How can this be tested?

Deploy a Dynakube that contains either one of the variables in the environment for the OneAgent and check if you get a warning.

## Checklist
